src/components/Map.ts

```diff
@@ -244,6 +249,24 @@ export class Map extends Component<MapProps, MapState> {
             && !(lat === 0 && lng === 0);
     }
 
+    private setCenter() {
+        if (this.props.defaultCenterLatitude && this.props.defaultCenterLongitude) {
+              this.setState({
+                  center: {
+                      lat: Number(this.props.defaultCenterLatitude),
+                      lng: Number(this.props.defaultCenterLongitude)
+                  }
+              });
+        } else if (this.props.defaultCenterAddress) {
+            this.getLocation(this.props.defaultCenterAddress, location =>
+                location ? this.setState({ center: location }) : this.setState({ center: this.defaultCenterLocation }));
+        } else {
+            if (this.bounds) {
```

Comment:
can combine `} else if (` in single line

Reply:
Done